app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1521,11 +1521,11 @@ private void setAndCalculatePokeSpamText(IVScanResult ivScanResult) {
                     pokeSpamCalculator.getTotalEvolvable(), pokeSpamCalculator.getEvolveRows(),
                     pokeSpamCalculator.getEvolveExtra());
 
-            if (totEvol<3) {
+            if (totEvol < 3) {
                 text = String.valueOf(pokeSpamCalculator.getTotalEvolvable());
-            } else if (totEvol>=3 && evolExtra==0) {
+            } else if (totEvol >= 3 && evolExtra == 0) {
                 text = textComplete;
-                text = text.substring(0,text.lastIndexOf("+")-1)+")";
+                text = text.substring(0,text.lastIndexOf("+") - 1) + ")";
```

Comment:
Here you are assigning the text variable twice, you can replace line 1527 and 1528 with:
`text = textComplete.substring(0, textComplete.lastIndexOf("+") - 1) + ")";`

Also, cutting a localized string and reformatting it is not a very good practise (other locales might not follow the same word order and thus brake your reformatting logic).
I suggest to add a new string resource!